rise/static/main.js

```diff
@@ -32,6 +32,7 @@ function configSlides() {
       slideNumber: true,
       start_slideshow_at: 'beginning',
       scroll: false,
+      center: true
```

Comment:
I am open to add this option, but I would prefer `center: false`, otherwise we are changing the `status quo`.